charmtools/git_version.py

```diff
@@ -88,3 +101,23 @@ def get_args(args=None):
         print('{version}{git}'.format(**version_info))
     else:
         print(version_info['version'])
+
+
+def get_args(args=None):
+    parser = argparse.ArgumentParser(description='Determine version')
+    parser.add_argument('--format', choices=['long', 'short', 'default'],
+                        default='default',
+                        help="Version format. Long includes git revision "
+                             "info. Default uses long if it's a pre-release.")
+    parser.add_argument('--build-charm-version', action='store_true')
+    args = parser.parse_args(args)
+
+    return args
+
+
+if __name__ == '__main__':
+    args = get_args()
+    if args.build_charm_version:
```

Comment:
Personally I would find ```print(_build_charm_version())``` slightly clearer that this script outputs something that is consumed elsewhere (as opposed to _build_charm_version() writing a file or something else etc.)